code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1471,6 +1471,8 @@ GLOBAL_LIST_EMPTY(aide_list)
 	var/turf/T = get_turf(target)
 	if(!T || timer > world.time)
 		return
+	if(isobj(target) && target.loc == user)//don't attack if we're clicking on our inventory
```

Comment:
You can check item_flags for ``IN_INVENTORY`` instead

Reply:
Oh, thats way nicer